crates/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -93,15 +96,31 @@ fn run_tests<B: Backend>(
             .expect("Failed to write to stdout");
         writer.flush().expect("Failed to flush writer");
 
-        match run_test(backend, &test_name, test_function, &context, show_output, compile_options) {
-            Ok(_) => {
+        match run_test(backend, test_function, &context, show_output, compile_options) {
```

Comment:
```suggestion
        match run_test(backend, &context, test_function, show_output, compile_options) {
```
To match up with above.